src/preprocessing/preprocessing_pass_context.h

```diff
@@ -57,6 +57,9 @@ class PreprocessingPassContext
     return d_circuitPropagator;
   }
 
+  /** Get the learned literal manager */
+  LearnedLiteralManager* getLearnedLiteralManager();
+
```

Comment:
Do we need this? If so: there is no implementation, and why would we return it as a pointer?

Reply:
Good catch, this was redundant.